### PR TITLE
use opensuse jekyll theme

### PR DESCRIPTION
**Pull request by guoyunhe**
so this repo is all about the content and we can reuse the theme in other sites.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -46,25 +55,24 @@ GEM
...
 PLATFORMS
   ruby
 
 DEPENDENCIES
   jekyll
-  jekyll-archives
-  jekyll-paginate
+  jekyll-theme-opensuse!
 
 BUNDLED WITH
```
Comment (hellcp) [inline Gemfile.lock:77]: Please remove BUNDLED WITH section

File: _config.yml [hunk 2]
```diff
@@ -1,13 +1,59 @@
...
 
+navbar_links:
+  - categories
+  -
+    icon: M6.94 14.036c-.233.624-.43 1.2-.606 1.783.96-.697 2.101-1.139 3.418-1.304 2.513-.314 4.746-1.973 5.876-4.058l-1.456-1.455 1.413-1.415 1-1.001c.43-.43.915-1.224 1.428-2.368-5.593.867-9.018 4.292-11.074 9.818zM17 9.001L18 10c-1 3-4 6-8 6.5-2.669.334-4.336 2.167-5.002 5.5H3C4 16 6 2 21 2c-1 2.997-1.998 4.996-2.997 5.997L17 9.001z
+    link: https://github.com/openSUSE/news-o-o#how-to-submit-a-new-post
+    text: New Post
+  -
+    icon: M3 17a4 4 0 0 1 4 4H3v-4zm0-7c6.075 0 11 4.925 11 11h-2a9 9 0 0 0-9-9v-2zm0-7c9.941 0 18 8.059 18 18h-2c0-8.837-7.163-16-16-16V3z
+    link: /feed/
```
Comment (hellcp) [inline _config.yml:23]: Please keep in  mind all of the current infrastructure redirects to feed.xml

File: Gemfile.lock (hunk 1 above)
Comment (guoyunhe) [reply Gemfile.lock]: Removed

File: _config.yml (hunk 2 above)
Comment (guoyunhe) [reply _config.yml:23]: Added `feed.xml` back for compatibility

**Pull request merged**